worldengine/cli/main.py

```diff
@@ -391,8 +391,8 @@ def main():
     if args.seed is not None:
         seed = int(args.seed)
     else:
-        seed = random.randint(0, 65535)#RNG initialization is done automatically
-    random.seed(seed)
+        seed = numpy.random.randint(0, 999999)  # RNG initialization is done automatically
```

Comment:
There are those multiplications for the different algorithms generation.py, max_int would be dangerous.
I didn't know about the platec restriction - if it were true, seeds that were manually put in should be restricted to that range, too!